### PR TITLE
Global exception handler

### PR DESCRIPTION
**Pull request by ihorromaniuk**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -47,6 +47,12 @@
 			<artifactId>mysql-connector-j</artifactId>
 			<version>9.1.0</version>
 		</dependency>
+		<dependency>
+			<groupId>org.hibernate.validator</groupId>
+			<artifactId>hibernate-validator</artifactId>
```
Comment (Elena-Bruyako) [inline pom.xml]: it's better to use `spring-boot-starter-validation`

File: src/main/java/core/basesyntax/bookstore/dto/book/CreateBookRequestDto.java [hunk 2]
```diff
@@ -1,11 +1,12 @@
 package core.basesyntax.bookstore.dto.book;
 
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
 
-public record CreateBookRequestDto(String title,
-                                   String author,
-                                   String isbn,
-                                   BigDecimal price,
+public record CreateBookRequestDto(@NotNull String title,
```
Comment (Elena-Bruyako) [inline src/main/java/core/basesyntax/bookstore/dto/book/CreateBookRequestDto.java]: for String use `@NotBlank`

```suggestion
public record CreateBookRequestDto(@NotBlank String title,
```

File: src/main/java/core/basesyntax/bookstore/dto/book/UpdateBookRequestDto.java [hunk 3]
```diff
@@ -1,11 +1,12 @@
 package core.basesyntax.bookstore.dto.book;
 
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
 
-public record UpdateBookRequestDto(String title,
-                                   String author,
-                                   String isbn,
-                                   BigDecimal price,
+public record UpdateBookRequestDto(@NotNull String title,
```
Comment (Elena-Bruyako) [inline src/main/java/core/basesyntax/bookstore/dto/book/UpdateBookRequestDto.java]: same here

**Review by Elena-Bruyako — CHANGES_REQUESTED** [review]
Good job! Let's improve your solution

File: src/main/java/core/basesyntax/bookstore/dto/book/CreateBookRequestDto.java [hunk 4]
```diff
@@ -1,11 +1,13 @@
...
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
 
-public record CreateBookRequestDto(String title,
-                                   String author,
-                                   String isbn,
-                                   BigDecimal price,
+public record CreateBookRequestDto(@NotBlank String title,
+                                   @NotNull String author,
+                                   @NotNull String isbn,
```
Comment (Elena-Bruyako) [inline src/main/java/core/basesyntax/bookstore/dto/book/CreateBookRequestDto.java]:
```suggestion
                                   @NotBlank String author,
                                   @NotBlank String isbn,
```

**Review by Elena-Bruyako — CHANGES_REQUESTED** [review]
see comment

**Review by liliia-ponomarenko — APPROVED** [review]
Awesome 😎

**Pull request merged**